L1Trigger/L1THGCal/interface/backend/HGCalHistoSeedingImpl.h

```diff
@@ -159,8 +161,10 @@ class HGCalHistoSeedingImpl {
   Navigator navigator_;
 
   static constexpr unsigned neighbour_weights_size_ = 9;
-  const double kROverZMin_ = 0.076;
-  const double kROverZMax_ = 0.58;
+  double kROverZMin_ = 0.076;
+  double kROverZMax_ = 0.58;
+  std::vector<double> vkROverZMin_;
+  std::vector<double> vkROverZMax_;
```

Comment:
The `k` prefix was meaning that these variables were constants (this is an usual convention). Since they are not constant anymore, the `k` prefix can be removed